utils/files.ts

```diff
@@ -184,11 +186,12 @@ export const collapseSmallPayments = (
     tempTxGroups[groupKey] = []
   }
 
-  const pushTx = (tx: TransactionsWithPaybuttonsAndPrices): void => {
+  const pushTx = (tx: TransactionsWithPaybuttonsAndPrices, groupKey: string): void => {
     const { timestamp, hash, address, amount } = tx
     const values = getTransactionValue(tx)
     const value = Number(values[currency])
     const rate = tx.prices.find(p => p.price.quoteId === QUOTE_IDS[currency.toUpperCase()])!.price.value
+    const buttonNames = groupKey.split('_').slice(2).join(' ; ')
```

Comment:
any reason for this spaces around the `;`? It's somewhat counter  intuitive and uncommon, so I'd take it out if there is no particular reason for it.